subworkflows/nf-core/fastq_qc_trim_filter_setstrandedness/main.nf

```diff
@@ -114,6 +116,16 @@ workflow FASTQ_QC_TRIM_FILTER_SETSTRANDEDNESS {
     ch_trim_read_count = Channel.empty()
     ch_multiqc_files   = Channel.empty()
 
+    //
+    // MODULE: Lint FastQ files
+    //
+    if(!skip_linting) {
+        FQ_LINT (
```

Comment:
Could you add the output of the linting module as output of this subworkflow please?